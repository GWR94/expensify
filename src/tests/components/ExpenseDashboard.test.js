import { shallow } from 'enzyme';
import React from 'react';
import ExpenseDashboard from '../../components/ExpenseDashboard';

test('should render expense dashboard and child components', () => {
    const wrapper = shallow(<ExpenseDashboard />);
    expect(wrapper).toMatchSnapshot();
});