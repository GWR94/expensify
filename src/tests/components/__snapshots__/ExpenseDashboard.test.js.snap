// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense dashboard and child components 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
